helm/superset/values.yaml

```diff
@@ -55,7 +55,9 @@ envFromSecrets: []
 extraEnv: {}
   # Extend timeout to allow long running queries.
   # GUNICORN_TIMEOUT: 300
-
+  # Increase the gunicorn worker amount, can improve performance drastically
+  # See: https://docs.gunicorn.org/en/stable/design.html#:~:text=Gunicorn%20should%20only%20need%204,workers%20to%20start%20off%20with.
+  # SERVER_WORKER_AMOUNT: 4
```

Comment:
would be great if you could add the rest of the variables: `SERVER_THREADS_AMOUNT`, `GUNICORN_KEEPALIVE`, `WORKER_MAX_REQUESTS `, `WORKER_MAX_REQUESTS_JITTER`, `SERVER_LIMIT_REQUEST_LINE`, `SERVER_LIMIT_REQUEST_FIELD_SIZE`

modify the link to just: https://docs.gunicorn.org/en/stable/design.html#how-many-workers

Reply:
I can't find `SERVER_WORKER_AMOUNT` anywhere in the Gunicorn docs

Reply:
@craig-rueda as you can see in [run-server.sh](https://github.com/apache/superset/blob/master/docker/run-server.sh#L26) the variable `SERVER_WORKER_AMOUNT` is introduced by Superset und used to set the flag `--workers` when starting gunicorn. It's not a gunicorn variable.

Reply:
@dpgaspar I added the variables and a link to the general documentation where users can find the settings.